portfolio-project-react/src/components/Details.jsx

```diff
@@ -0,0 +1,31 @@
+const Details = () => {
+  return (
+    <div className='details-section'>
+      <h1 class='title-about'>About</h1>
+      <p id='intro-data'>
+        I am a highly experienced Full Stack Web Developer with a proven track
```

Comment:
All of your static content, that includes strings and meaningful numeric values should be inside a newly created CONSTANTS file. 

Each CONSTANTS file should be limited to its component, if the the static content is being used other than the component than it should be placed in the root CONSTANTS file.